### DISCUSSION TITLE
part-ii/2018-89/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#213] [Question comments]
Part II 2018 Logic and Set Theory: Paper 1, Section II, G 

https://questions.tripos.org/part-ii/2018-89/

**Comment by dropdbassett** [comment]
<details><summary>Give an example of ordinals such that...</summary>

$\alpha = \omega 2$ and $\beta = \omega 3$ work; the $\omega$-th powers are the sup of the natural powers, which we may use associativity to show that $\alpha^n = \omega^n 2$ and $\beta^n = \omega^n 3$. So $\alpha^\omega = \beta^\omega = \omega^\omega$.
</details>

<details><summary>Give an example to show that we need not have ...</summary>

As before $(\omega 2)^\omega = \omega^\omega$. But $\omega^\omega 2^\omega = \omega^{\omega + 1}$.
</details>

<details><summary>For which ordinals do we have ...</summary>

For $\alpha \ge 2$, a simple induction shows we always have $\alpha^\beta \ge \beta$. So any $\alpha \ge 2$ works.
</details>

<details><summary>And for which do we have ...</summary>

It holds for $\alpha \ge \omega_2$ by using the first part of the question. 

If $\alpha < \omega_2$, then $\mathrm{card}(\alpha) \le \aleph_1$. We prove by induction that $\mathrm{card}\left(\alpha^\gamma\right) \le \aleph_1$ for all $\gamma \le \omega_1$. 
- This clearly holds for $\gamma = 0$. 
- If $\gamma = \beta^+$ then $\alpha^\gamma = \alpha^\beta \alpha$ is a well-ordering on $\alpha^\beta \times \alpha$, by considering the synthetic definition of multiplication. But now by induction  $\mathrm{card}(\alpha^\beta \times \alpha) \le \aleph_1 \aleph_1 = \aleph_1$.
- If $\gamma$ is a non-zero limit, then we are taking a union of at most $\aleph_1$ sets of cardinality at most $\aleph_1$, and hence we may inject them into $\aleph_1 \aleph_1 = \aleph_1$.

Thus the inequality doesn't hold for any $\alpha < \omega_2$.
</details>
